benchmarks/distributed/rpc/rl/agent.py

```diff
@@ -0,0 +1,150 @@
+from functools import reduce
+import time
+import threading
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+
+import torch.distributed.rpc as rpc
+from torch.distributed.rpc import RRef, remote
+from torch.distributions import Categorical
+
+from torch import autograd
+
+OBSERVER_NAME = "observer{}"
+gamma = 0.99
+
+torch.autograd.set_detect_anomaly(True)
+
+
+class Policy(nn.Module):
+    def __init__(self, state_size, nlayers, out_features, batch=True):
+        torch.autograd.set_detect_anomaly(True)
+        super(Policy, self).__init__()
+        self.in_features = reduce((lambda x, y: x*y), state_size)
+
+        self.model = nn.Sequential(
+            nn.Flatten(1, -1),
+            nn.Linear(self.in_features, out_features),
+            * [nn.Linear(out_features, out_features) for _ in range(nlayers)]
+        )
+        self.dim = 0
+
+    def forward(self, x):
+        action_scores = self.model(x)
+        return F.softmax(action_scores, dim=self.dim)
```

Comment:
I only reviewed in the `Policy` class, please apply the same standard to all other code as well, i.e.:

1. remove unused vars/methods
2. add comments to explain its behavior and args
3. check if there is any concept that does not belong to it
4. remove debugging vestige. 
5. fix indents and empty lines